File: TimedTrials/Models/Trial.cs
using System;

namespace TimedTrials.Models
{
    public class Trial
    {
        public int Id { get; set; }
        public int TrialDuration { get; set; }
        public DateTime TrialExpirationDate { get; set; }
        public decimal SubscriptionPrice { get; set; }
        public int WebsiteId { get; set; }
        public Website Website { get; set; }
    }
}

File: TimedTrials/client/src/components/Trial.js
import React from "react";
import { Card, CardBody, CardTitle } from "reactstrap";

const Trial = ({ trial }) => {
  return (
    <Card>
      <div>
        <CardBody>
          <p>
            <a className="webLink" href={trial.website.url}>
              {trial.website.name}
            </a>
            <hr />
          </p>
          <h6>Subscription fee after trial ends:</h6>
          <p>${trial.subscriptionPrice} per month</p>
          <hr />
          <h6>Trial offer ends on:</h6>
          <p>{trial.trialExpirationDate.substr(0, 10)}</p>
          <hr />
          <h6>Trial duration:</h6>
          <p>{trial.trialDuration} Days</p>
        </CardBody>
      </div>
    </Card>
  );
};

export default Trial;
